subsys/testsuite/ztest/include/zephyr/ztest_test_new.h

```diff
@@ -263,7 +263,7 @@ void ztest_test_pass(void);
 void ztest_test_skip(void);
 
 #define Z_TEST(suite, fn, t_options, use_fixture)                                                  \
-	struct ztest_unit_test_stats UTIL_CAT(z_ztest_unit_test_stats_, fn);      \
+	struct ztest_unit_test_stats z_ztest_unit_test_stats_##suite##_##fn;             \
 	static void _##suite##_##fn##_wrapper(void *data);                                         \
```

Comment:
please keep the trailing backslashes nicely aligned

Reply:
done